08-rails/http-verbs-crud-and-idempotency.md

```diff
@@ -13,7 +13,7 @@ Kinda like this:
 
```

Comment:
We should use `/book` or `/books` here as well, and change the associated text to talk about what the result of a `GET` request to that route would return.